base_tier_validation_escalate/tests/test_tier_validation.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2018 ForgeFlow S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo.tests import Form, common
+
+from .common import setup_test_model, teardown_test_model
```

Comment:
I would change this to `from odoo.addons.base_tier_validation.tests.common import setup_test_model, teardown_test_model` and remove the common file in this module.

Reply:
Thank you and fixed.